cmd/kubeadm/app/features/features.go

```diff
@@ -102,15 +102,6 @@ func Supports(featureList FeatureList, featureName string) bool {
 	return false
 }
 
-// Keys returns a slice of feature names for a given feature set
-func Keys(featureList FeatureList) []string {
```

Comment:
removing this seems like should be in another commit instead of here, fine as is.